module_programming_ast/module_programming_ast/__main__.py

```diff
@@ -73,71 +75,23 @@ def process_incoming_feedback(exercise: Exercise, submission: Submission, feedba
 def suggest_feedback(exercise: Exercise, submission: Submission, module_config: Configuration) -> List[Feedback]:
     logger.info("suggest_feedback: Suggestions for submission %d of exercise %d were requested", submission.id, exercise.id)
     # Do something with the submission and return a list of feedback
+    # ThemisML currently only works with Java
+    if exercise.programming_language.lower() != "java":
+        logger.info("ThemisML only works with Java. Returning no suggestions.")
```

Comment:
Does this still make sense?
```suggestion
        logger.info("AST Module only works with Java. Returning no suggestions.")
```